sympy/matrices/expressions/tests/test_diagonal.py

```diff
@@ -9,9 +9,18 @@
 d = DiagonalOf(X)
```

Comment:
please remove the line with `x = ` and keep the `X =` line. Then change the `x` in `D = ` to `X` and delete the first line in the `test_DiagonalMatrix` function.